charts/gitlab-ci-pipelines-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 apiVersion: v2
 name: gitlab-ci-pipelines-exporter
-version: 0.2.12
+version: 0.3.1
```

Comment:
would you mind bumping to `0.2.13` instead? :)

Reply:
Done!